api/schemas.py

```diff
@@ -12,12 +12,15 @@ class FieldSchema(Schema):
     @pre_dump
     def serialize_input_options(self, data, many):
         """Convert input_options from bytes obj to dict."""
-        if data.input_options:
-            input_options = data.input_options.decode().replace("'", '"')
-            data.input_options = json.loads(input_options)
-            return data
-        else:
-            return data
+        try:
+            if data.input_options and isinstance(data.input_options, dict) is False:
```

Comment:
use `not isinstance` instead of `is false`

Reply:
Addressed with [af2689e](https://github.com/wikimedia/toolhunt/pull/55/commits/af2689e1124495e48753a69ac5c25a956f7c09a0)